### PR TITLE
Contributor Experience Charter

### PR DESCRIPTION
**Pull request by parispittman**
Trying this charter thing out again since the mistake during the recent outage merged to master. Ref #2810.  

Incorporated most of @spiffxp's review from #2810 except the GitHub management bullets which I'd rather have @cblecker weigh in directly about. Expecting another edit.

### CODE REVIEW

File: sig-contributor-experience/charter.md [hunk 1]
```diff
@@ -1,58 +1,141 @@
...
+  - Retrospective moderation for other SIGs upon request
+  - Other events, like other SIG face to face events, upon request and consideration
+- Strategize, build, and execute on scalable [mentoring programs] for all contributor levels. These may include:
+  - [Google Summer of Code]
+  - [Outreachy]
+  - [Meet Our Contributors]
+  - [Group Mentoring - WIP]
+  - [The 1:1 Hour - WIP]
+  - Speed Mentoring at KubeCon
+- Create best practices, trainings, policies, and [moderation] of Kubernetes communication platforms, the transparency vehicles of information throughout the project. This includes user guidelines when the contributor is the user of the platform. Moderation is an always on, never off service that we supply with trusted contributors spanning time zones. This includes immediate action when dealing with code of conduct issues in a public space that are defined below:
```
Comment (cblecker) [inline sig-contributor-experience/charter.md]: This is pretty long for a point form list, and mixes both owning the platforms, and doing moderation on them. Perhaps this can be shortened or split up?

Also, we (via the github admin team) would handle moderation on GitHub too, where that's needed.

Comment (cblecker) [reply sig-contributor-experience/charter.md]: much clearer! 👍

File: sig-contributor-experience/charter.md [hunk 2]
```diff
@@ -1,58 +1,141 @@
...
+  - routine GitHub management tasks, including but not limited to: org membership, org permissions, repo creation/administration.
+  - "Org Owner" GitHub permissions, and grant/limit these privileges accordingly
+  - bot accounts, service accounts, webhooks, and third-party integrations for all communication platforms that we support but most importantly, GitHub.
+- Establishing a "GitHub Administration team" that will oversee the execution of GitHub management tasks: inviting new members to the org, creating repos, executing moderation decisions, auditing permissions.
+- Work with other SIGs and interested parties in the project to execute GitHub tasks where required
+-Own and execute events that are targeted to the Kubernetes contributor community, including:
+  - The weekly [Kubernetes Community meeting]
+  - [Contributor Summit(s)]
+  - SIG Contributor Experience face to face meetings
+  - [Steering Committee elections] (though we do not own policy creation, see 'out of scope' below)
```
[the remark this answers is not in the document]
Comment (parispittman) [reply sig-contributor-experience/charter.md]: @spiffxp - do you want to shed some light on this from a steering perspective? 
I think contribex should own the process as it's a contributor only event and we should influence events that are solely for contributors.

File: sig-contributor-experience/charter.md [hunk 3]
```diff
@@ -1,58 +1,141 @@
...
+
+We do this by listening - whether it’s through our roadshows to SIG meetings, surveys, data, or [GitHub issues], we take in the feedback and turn it into our [project list]. We build a welcoming and inclusive community of contributors by giving them places to be heard and productive.
+
+### In scope
+
+#### Code, Binaries and Services
+
+- Establish policies, standards, and procedures:
+  - routine GitHub management tasks, including but not limited to: org membership, org permissions, repo creation/administration.
+  - "Org Owner" GitHub permissions, and grant/limit these privileges accordingly
```
[the remark this answers is not in the document]
Comment (parispittman) [reply sig-contributor-experience/charter.md]: removed.

File: sig-contributor-experience/charter.md [hunk 4]
```diff
@@ -1,58 +1,141 @@
...
+  - bot accounts, service accounts, webhooks, and third-party integrations for all communication platforms that we support but most importantly, GitHub.
+- Establishing a "GitHub Administration team" that will oversee the execution of GitHub management tasks: inviting new members to the org, creating repos, executing moderation decisions, auditing permissions.
```
[the remark this answers is not in the document]
Comment (parispittman) [reply sig-contributor-experience/charter.md]: removed.

File: sig-contributor-experience/charter.md [hunk 5]
```diff
@@ -1,58 +1,141 @@
...
+- Work with other SIGs and interested parties in the project to execute GitHub tasks where required
+-Own and execute events that are targeted to the Kubernetes contributor community, including:
```
[the remark this answers is not in the document]
Comment (parispittman) [reply sig-contributor-experience/charter.md]: removed

**Comment by parispittman** [comment]
@cblecker - i think this leaves creation of subprojects and steering committee election process as the last remaining items to discuss/clear up. Any other feedback?

@spiffxp - we are close!

**Comment by cblecker** [comment]
I'm good with all of @spiffxp's feedback above.

File: sig-contributor-experience/charter.md [hunk 6]
```diff
@@ -1,58 +1,142 @@
...
+
+Ensuring that technical changes from subprojects follow the process change communication guidelines listed above.
+
+### Deviations from sig-governance
+Six months after this charter is first ratified, it MUST be reviewed and re-approved by the SIG in order to evaluate the assumptions made in its initial drafting.
+
+### Subproject Creation - TODO
+Pick one:
+SIG Technical Leads
+Federation of Subprojects
```
[the remark this answers is not in the document]
Comment (cblecker) [reply sig-contributor-experience/charter.md]: I think in our case, Chairs + TLs is the best option. Still gives a wide perspective, but in practice, we've never needed a tie break.. consensus of the body is always best.

File: sig-contributor-experience/charter.md [hunk 7]
```diff
@@ -1,58 +1,138 @@
...
+- Encourage automation to improve productivity for contributors where it makes sense and consult with SIG Testing if automation is covering GitHub workflows.
+- Research other OSS projects and consult with their leaders about contributor experience topics to make sure we are always delivering value to our contributors.
+
+#### Cross-cutting and Externally Facing Processes
+
+We cross-cut all SIGs and WGs to deliver the following processes:
+
+- Deploying Changes:
+  When implementing policy changes we strive to balance  responding quickly to the needs of the community and ensuring a disruption-free experience for project contributors. As such, the amount of notice we provide and the amount of consensus we seek is driven by our estimation of risk. We don't measure risk objectively at this time, but estimate it based on these parameters:
+  - Low risk changes impact a small number (<4) of SIGs, WGs, or repos, do not break existing contributor workflows, and are easy to roll back. When implementing low risk changes we:
```
Comment (idealhack) [inline sig-contributor-experience/charter.md]:
```suggestion
  - Low-risk changes impact a small number (<4) of SIGs, WGs, or repos, do not break existing contributor workflows, and are easy to roll back. When implementing low-risk changes we:
```

File: sig-contributor-experience/charter.md [hunk 8]
```diff
@@ -1,58 +1,138 @@
...
+
+If we need funding for any reason, we approach [CNCF].
+CNCF in many of the noted cases above, contributes funding to our platforms, processes, and/or programs. They do not play a day-to-day operations role and have bestowed that to our community to run as we see fit. Since they do fund some of our initatives, this means that they hold owner account privileges on certain platforms like Zoom and Slack. In these cases, such as Slack, there are at least two Contributor Experience [communication] subproject OWNERs listed as admins.
+
+### Out of scope
+
+- Code for the testing and CI infrastructure - that’s SIG Testing
+- [kubernetes/community]  ownership of folders for KEPs and Design Proposals. Members are to follow those folders owners files and SIG leadership for the specific issue/PR in question.
+- User community management. We hold office hours because contributors are a large portion of the volunteers that run that program.
+- The contributor experience for repos not included in the Kubernetes [associated repositories]list.
```
Comment (idealhack) [inline sig-contributor-experience/charter.md]:
```suggestion
- The contributor experience for repos not included in the Kubernetes [associated repositories] list.
```

File: sig-contributor-experience/charter.md [hunk 9]
```diff
@@ -1,58 +1,138 @@
...
+
+We cross-cut all SIGs and WGs to deliver the following processes:
+
+- Deploying Changes:
+  When implementing policy changes we strive to balance  responding quickly to the needs of the community and ensuring a disruption-free experience for project contributors. As such, the amount of notice we provide and the amount of consensus we seek is driven by our estimation of risk. We don't measure risk objectively at this time, but estimate it based on these parameters:
+  - Low risk changes impact a small number (<4) of SIGs, WGs, or repos, do not break existing contributor workflows, and are easy to roll back. When implementing low risk changes we:
+    - Socialize on kubernetes-sig-contribex@googlegroups.com and on our weekly update calls
+    - We will go to each lead, their mailing lists, slack channel, and/or their update meetings and ask for feedback and a [lazy consensus] process. We will follow up with a post to [kubernetes-dev@]googlegroups.com mailing list
+  - High risk changes impact a large number (<4) of SIGs, WGs, or repos, break existing contributor workflows, and are not easy to roll back. When implementing high risk changes we:
```
Comment (idealhack) [inline sig-contributor-experience/charter.md]:
```suggestion
  - High-risk changes impact a large number (<4) of SIGs, WGs, or repos, break existing contributor workflows, and are not easy to roll back. When implementing high-risk changes we:
```

File: sig-contributor-experience/charter.md [hunk 10]
```diff
@@ -1,58 +1,138 @@
...
+    - [kubernetes-sig-contribex@]googlegroups.com
+    - sig-leads@googlegroups.com
+    - [kubernetes-dev@]googlegroups.com with the GitHub issue link including the subject [NOTICE]: $announcement
+    - We will also announce it at the weekly Kubernetes Community Meeting on Thursdays
+- Depending on how wide of an ecosystem change this is, we may also slack, blog, tweet, and use other channels to get the word out.
+- Our standard time box is 72 business hours; however, there may be situations where we need to act quickly but the time period will always be clear and upfront.
+
+If we need funding for any reason, we approach [CNCF].
+CNCF in many of the noted cases above, contributes funding to our platforms, processes, and/or programs. They do not play a day-to-day operations role and have bestowed that to our community to run as we see fit. Since they do fund some of our initatives, this means that they hold owner account privileges on certain platforms like Zoom and Slack. In these cases, such as Slack, there are at least two Contributor Experience [communication] subproject OWNERs listed as admins.
```
Comment (idealhack) [inline sig-contributor-experience/charter.md]:
```suggestion
CNCF in many of the noted cases above, contributes funding to our platforms, processes, and/or programs. They do not play a day-to-day operations role and have bestowed that to our community to run as we see fit. Since they do fund some of our initiatives, this means that they hold owner account privileges on certain platforms like Zoom and Slack. In these cases, such as Slack, there are at least two Contributor Experience [communication] subproject OWNERs listed as admins.
```

File: sig-contributor-experience/charter.md [hunk 11]
```diff
@@ -1,58 +1,138 @@
...
+  - engagement on project platforms that we manage
+- Encourage automation to improve productivity for contributors where it makes sense and consult with SIG Testing if automation is covering GitHub workflows.
+- Research other OSS projects and consult with their leaders about contributor experience topics to make sure we are always delivering value to our contributors.
+
+#### Cross-cutting and Externally Facing Processes
+
+We cross-cut all SIGs and WGs to deliver the following processes:
+
+- Deploying Changes:
+  When implementing policy changes we strive to balance  responding quickly to the needs of the community and ensuring a disruption-free experience for project contributors. As such, the amount of notice we provide and the amount of consensus we seek is driven by our estimation of risk. We don't measure risk objectively at this time, but estimate it based on these parameters:
```
Comment (idealhack) [inline sig-contributor-experience/charter.md]:
```suggestion
  When implementing policy changes we strive to balance responding quickly to the needs of the community and ensuring a disruption-free experience for project contributors. As such, the amount of notice we provide and the amount of consensus we seek is driven by our estimation of risk. We don't measure risk objectively at this time, but estimate it based on these parameters:
```

File: sig-contributor-experience/charter.md [hunk 12]
```diff
@@ -1,58 +1,138 @@
...
+  - Low risk changes impact a small number (<4) of SIGs, WGs, or repos, do not break existing contributor workflows, and are easy to roll back. When implementing low risk changes we:
+    - Socialize on kubernetes-sig-contribex@googlegroups.com and on our weekly update calls
```
Comment (idealhack) [inline sig-contributor-experience/charter.md]:
```suggestion
    - Socialize on kubernetes-sig-contribex@googlegroups.com and our weekly update calls
```

File: sig-contributor-experience/charter.md [hunk 13]
```diff
@@ -1,58 +1,138 @@
...
+    - We will go to each lead, their mailing lists, slack channel, and/or their update meetings and ask for feedback and a [lazy consensus] process. We will follow up with a post to [kubernetes-dev@]googlegroups.com mailing list
+  - High risk changes impact a large number (<4) of SIGs, WGs, or repos, break existing contributor workflows, and are not easy to roll back. When implementing high risk changes we:
+    - Socialize on kubernetes-sig-contribex@googlegroups.com and on our weekly update calls
+    - Seek [lazy consensus] with a time box of at least 72 business hours with a GitHub issue link (or proposal if not applicable) to the following mailing lists:
+    - [kubernetes-sig-contribex@]googlegroups.com
```
Comment (idealhack) [inline sig-contributor-experience/charter.md]:
```suggestion
        - [kubernetes-sig-contribex@]googlegroups.com
```

File: sig-contributor-experience/charter.md [hunk 14]
```diff
@@ -1,58 +1,138 @@
...
+    - sig-leads@googlegroups.com
```
Comment (idealhack) [inline sig-contributor-experience/charter.md]:
```suggestion
        - sig-leads@googlegroups.com
```

File: sig-contributor-experience/charter.md [hunk 15]
```diff
@@ -1,58 +1,138 @@
...
+    - [kubernetes-dev@]googlegroups.com with the GitHub issue link including the subject [NOTICE]: $announcement
```
Comment (idealhack) [inline sig-contributor-experience/charter.md]:
```suggestion
        - [kubernetes-dev@]googlegroups.com with the GitHub issue link including the subject [NOTICE]: $announcement
```

File: sig-contributor-experience/charter.md [hunk 16]
```diff
@@ -1,58 +1,138 @@
...
+
+- Deploying Changes:
+  When implementing policy changes we strive to balance  responding quickly to the needs of the community and ensuring a disruption-free experience for project contributors. As such, the amount of notice we provide and the amount of consensus we seek is driven by our estimation of risk. We don't measure risk objectively at this time, but estimate it based on these parameters:
+  - Low risk changes impact a small number (<4) of SIGs, WGs, or repos, do not break existing contributor workflows, and are easy to roll back. When implementing low risk changes we:
+    - Socialize on kubernetes-sig-contribex@googlegroups.com and on our weekly update calls
+    - We will go to each lead, their mailing lists, slack channel, and/or their update meetings and ask for feedback and a [lazy consensus] process. We will follow up with a post to [kubernetes-dev@]googlegroups.com mailing list
+  - High risk changes impact a large number (<4) of SIGs, WGs, or repos, break existing contributor workflows, and are not easy to roll back. When implementing high risk changes we:
+    - Socialize on kubernetes-sig-contribex@googlegroups.com and on our weekly update calls
```
Comment (idealhack) [inline sig-contributor-experience/charter.md]: the same with low-risk ones?

if so:
```suggestion
    - Socialize on kubernetes-sig-contribex@googlegroups.com and our weekly update calls
```

File: sig-contributor-experience/charter.md (hunk 15 above)
Comment (idealhack) [reply sig-contributor-experience/charter.md]: indent these three mailing lists

File: sig-contributor-experience/charter.md [hunk 17]
```diff
@@ -1,58 +1,142 @@
...
+
+## Scope
+
+The [Contributor Experience Special Interest Group] (SIG) is responsible for improving the experience of those who upstream contribute to the Kubernetes project. We do this by creating, and maintaining programs and processes that promote community health and reduce project friction, while retiring those programs and processes that don't. Being conscientious of our contributor base is critical to scaling the project, growing the ecosystem, and helping the project succeed.
+
+We do this by listening - whether it’s through our roadshows to SIG meetings, surveys, data, or [GitHub issues], we take in the feedback and turn it into our [project list]. We build a welcoming and inclusive community of contributors by giving them places to be heard and productive.
+
+### In scope
+
+#### Code, Binaries and Services
```
[the remark this answers is not in the document]
Comment (parispittman) [reply sig-contributor-experience/charter.md]: fixed

File: sig-contributor-experience/charter.md [hunk 18]
```diff
@@ -1,58 +1,142 @@
...
+- Perform issue triage on and maintain the [kubernetes/community] repository.  
+- Help SIGs with being as transparent and open as possible through creating best practices, guidelines, and general administration of YouTube, Zoom, and our mailing lists where applicable
+- Assist SIGs/WG Chairs and Technical Leads with organizational management operations as laid out in the [sig-governance] doc
+- Distribute contributor related news on various Kubernetes channels, including Cloud Native Compute Foundation [CNCF] for posting blogs, social media, and other platforms as needed.
+- Establish and share metrics to measure project health, community health, and general trends, including:
+  - ongoing work with the CNCF to improve [DevStats]
+  - the contributor experience survey(s)
+  - engagement on project platforms that we manage
+- Encourage automation to improve productivity for contributors where it makes sense and consult with SIG Testing if automation is covering GitHub workflows.
```
[the remark this answers is not in the document]
Comment (parispittman) [reply sig-contributor-experience/charter.md]: fixed

File: sig-contributor-experience/charter.md (hunk 7 above)
Comment (parispittman) [reply sig-contributor-experience/charter.md]: fixed

File: sig-contributor-experience/charter.md (hunk 8 above)
Comment (parispittman) [reply sig-contributor-experience/charter.md]: fixed

File: sig-contributor-experience/charter.md (hunk 9 above)
Comment (parispittman) [reply sig-contributor-experience/charter.md]: fixed

File: sig-contributor-experience/charter.md (hunk 10 above)
Comment (parispittman) [reply sig-contributor-experience/charter.md]: fixed

File: sig-contributor-experience/charter.md (hunk 11 above)
Comment (parispittman) [reply sig-contributor-experience/charter.md]: fixed

File: sig-contributor-experience/charter.md (hunk 12 above)
Comment (parispittman) [reply sig-contributor-experience/charter.md]: fixed

File: sig-contributor-experience/charter.md (hunk 13 above)
Comment (parispittman) [reply sig-contributor-experience/charter.md]: fixed

File: sig-contributor-experience/charter.md (hunk 14 above)
Comment (parispittman) [reply sig-contributor-experience/charter.md]: fixed

File: sig-contributor-experience/charter.md (hunk 15 above)
Comment (parispittman) [reply sig-contributor-experience/charter.md]: fixed

File: sig-contributor-experience/charter.md (hunk 16 above)
Comment (parispittman) [reply sig-contributor-experience/charter.md]: fixed

**Comment by parispittman** [comment]
@sarahnovotny

File: sig-contributor-experience/charter.md [hunk 19]
```diff
@@ -1,58 +1,143 @@
...
+[Meet Our Contributors]:  https://git.k8s.io/community/mentoring/meet-our-contributors.md
+[Group Mentoring - WIP]:  https://git.k8s.io/community/mentoring/group-mentoring.md
+[The 1:1 Hour - WIP]: https://git.k8s.io/community/mentoring/the1-on-1hour.md
+[kubernetes/community]: https://git.k8s.io/community/
+[Contributor Summit(s)]: https://git.k8s.io/community/events/2018/12-contributor-summit
+[contributor summits]: https://git.k8s.io/community/events/2018/12-contributor-summit
+[DevStats]: https://k8s.cncf.devstats.io
+[kubernetes-sig-contribex@]: https://groups.google.com/forum/#!forum/kubernetes-sig-contribex
+[kubernetes blog]: https://www.kubernetes.io/blog
+[GitHub]: https://git.k8s.io/community/github-management
```
Comment (nikhita) [inline sig-contributor-experience/charter.md]: nit: let's call this github-management instead of just `GitHub`? :grimacing:

File: sig-contributor-experience/charter.md [hunk 20]
```diff
@@ -1,58 +1,143 @@
...
+[communication]: https://git.k8s.io/community/communication
+[CNCF]: https://cncf.io
+[GitHub issues]: https://github.com/kubernetes/community/issues
+[project list]: https://github.com/orgs/kubernetes/projects/1
+[Kubernetes Community meeting]: https://git.k8s.io/community/
+[mentoring programs]: https://git.k8s.io/community/mentoring
+[Steering Committee elections]: https://git.k8s.io/community/
```
Comment (nikhita) [inline sig-contributor-experience/charter.md]: I think this should point to https://git.k8s.io/community/events/elections

File: sig-contributor-experience/charter.md [hunk 21]
```diff
@@ -1,58 +1,143 @@
...
+  - Retrospective moderation for other SIGs upon request
+  - Other events, like other SIG face to face events, upon request and consideration
+- Strategize, build, and execute on scalable [mentoring programs] for all contributor levels. These may include:
+  - [Google Summer of Code]
+  - [Outreachy]
+  - [Meet Our Contributors]
+  - [Group Mentoring - WIP]
+  - [The 1:1 Hour - WIP]
+  - Speed Mentoring at KubeCon
+We supply the moderation teams that span time zones for 24/7 coverage. This includes immediate action when dealing with code of conduct issues in a public spaces
```
Comment (nikhita) [inline sig-contributor-experience/charter.md]: - Is this for the speed mentoring session or in general? If in general, this needs to go as a sub-point under moderation, or else it renders with the Speed Mentoring point.
- Maybe we can rephrase this as `We supply moderation teams spanning multiple time zones for 24/7 coverage.`?
- s/a public spaces/public places.

File: sig-contributor-experience/charter.md [hunk 22]
```diff
@@ -1,58 +1,143 @@
...
+- Help onboard new contributors and current into the culture, workflow, and CI of our contributor experience through the [contributor guide], other related documentation, [contributor summits] and programs tailored to onboarding.
```
Comment (nikhita) [inline sig-contributor-experience/charter.md]: nit: s/new contributors and current/new and current contributors

File: sig-contributor-experience/charter.md [hunk 23]
```diff
@@ -1,58 +1,143 @@
...
+- Perform issue triage on and maintain the [kubernetes/community] repository.  
+- Help SIGs with being as transparent and open as possible through creating best practices, guidelines, and general administration of YouTube, Zoom, and our mailing lists where applicable
+- Assist SIGs/WG Chairs and Technical Leads with organizational management operations as laid out in the [sig-governance] doc
+- Distribute contributor related news on various Kubernetes channels, including Cloud Native Compute Foundation [CNCF] for posting blogs, social media, and other platforms as needed.
```
Comment (nikhita) [inline sig-contributor-experience/charter.md]: nit: s/[CNCF]/([CNCF]) ... i.e. CNCF in brackets

File: sig-contributor-experience/charter.md [hunk 24]
```diff
@@ -1,58 +1,143 @@
...
+
+We cross-cut all SIGs and WGs to deliver the following processes:
+
+- Deploying Changes:
+  When implementing policy changes we strive to balance responding quickly to the needs of the community and ensuring a disruption-free experience for project contributors. As such, the amount of notice we provide and the amount of consensus we seek is driven by our estimation of risk. We don't measure risk objectively at this time, but estimate it based on these parameters:
+  - Low-risk changes impact a small number (<4) of SIGs, WGs, or repos, do not break existing contributor workflows, and are easy to roll back. When implementing low-risk changes we:
+    - Socialize on kubernetes-sig-contribex@googlegroups.com and our weekly update calls
+    - We will go to each lead, their mailing lists, slack channel, and/or their update meetings and ask for feedback and a [lazy consensus] process. We will follow up with a post to [kubernetes-dev@]googlegroups.com mailing list
+  - High-risk changes impact a large number (<4) of SIGs, WGs, or repos, break existing contributor workflows, and are not easy to roll back. When implementing high-risk changes we:
```
Comment (nikhita) [inline sig-contributor-experience/charter.md]: `>4` instead of `<4`

File: sig-contributor-experience/charter.md [hunk 25]
```diff
@@ -1,58 +1,143 @@
...
+[contributor summits]: https://git.k8s.io/community/events/2018/12-contributor-summit
+[DevStats]: https://k8s.cncf.devstats.io
+[kubernetes-sig-contribex@]: https://groups.google.com/forum/#!forum/kubernetes-sig-contribex
+[kubernetes blog]: https://www.kubernetes.io/blog
+[GitHub]: https://git.k8s.io/community/github-management
+[communication]: https://git.k8s.io/community/communication
+[CNCF]: https://cncf.io
+[GitHub issues]: https://github.com/kubernetes/community/issues
+[project list]: https://github.com/orgs/kubernetes/projects/1
+[Kubernetes Community meeting]: https://git.k8s.io/community/
```
Comment (nikhita) [inline sig-contributor-experience/charter.md]: This could point to communication#weekly-meeting

File: sig-contributor-experience/charter.md [hunk 26]
```diff
@@ -1,58 +1,143 @@
...
+
+If we need funding for any reason, we approach [CNCF].
+CNCF in many of the noted cases above, contributes funding to our platforms, processes, and/or programs. They do not play a day-to-day operations role and have bestowed that to our community to run as we see fit. Since they do fund some of our initiatives, this means that they hold owner account privileges on certain platforms like Zoom and Slack. In these cases, such as Slack, there are at least two Contributor Experience [communication] subproject OWNERs listed as admins.
+
+### Out of scope
+
+- Code for the testing and CI infrastructure - that’s SIG Testing
+- [kubernetes/community]  ownership of folders for KEPs and Design Proposals. Members are to follow those folders owners files and SIG leadership for the specific issue/PR in question.
+- User community management. We hold office hours because contributors are a large portion of the volunteers that run that program.
+- The contributor experience for repos not included in the Kubernetes [associated repositories] list.
```
Comment (nikhita) [inline sig-contributor-experience/charter.md]: This needs a link to be mentioned below in the doc.

https://git.k8s.io/community/github-management/kubernetes-repositories.md#associated-repositories

File: sig-contributor-experience/charter.md [hunk 27]
```diff
@@ -1,58 +1,143 @@
...
+
+## Roles and Organization Management
+
+This sig adheres to the Roles and Organization Management outlined in [sig-governance]
+and opts-in to updates and modifications to [sig-governance].
+
+
+### Additional responsibilities of Chairs
+
+Chairs SHOULD plan at least one face to face Contributor Experience meeting per year
```
Comment (nikhita) [inline sig-contributor-experience/charter.md]: :heart_eyes:

File: sig-contributor-experience/charter.md (hunk 19 above)
Comment (parispittman) [reply sig-contributor-experience/charter.md]: fixed

File: sig-contributor-experience/charter.md (hunk 20 above)
Comment (parispittman) [reply sig-contributor-experience/charter.md]: fixed

File: sig-contributor-experience/charter.md (hunk 21 above)
Comment (parispittman) [reply sig-contributor-experience/charter.md]: fixed

File: sig-contributor-experience/charter.md (hunk 22 above)
Comment (parispittman) [reply sig-contributor-experience/charter.md]: fixed

File: sig-contributor-experience/charter.md (hunk 23 above)
Comment (parispittman) [reply sig-contributor-experience/charter.md]: fixed

File: sig-contributor-experience/charter.md (hunk 24 above)
Comment (parispittman) [reply sig-contributor-experience/charter.md]: fixed

File: sig-contributor-experience/charter.md (hunk 25 above)
Comment (parispittman) [reply sig-contributor-experience/charter.md]: fixed

**Comment by dims** [comment]
/lgtm
/hold

Please release hold when you wish. Charter looks good!

**Comment by parispittman** [comment]
waiting for @sarahnovotny's final go and then I'll release the hold
